app/controllers/anonymous_feedback/export_requests_controller.rb

```diff
@@ -48,8 +51,22 @@ def clean_paths
     end
   end
 
+  def saved_paths
+    @saved_paths ||= anonymous_feedback_params[:paths].present? ? Support::Requests::Anonymous::Paths.find(anonymous_feedback_params[:paths].first) : nil
```

Comment:
I'm struggling to understand what the `.first` is doing here. I can see a similar occurrence in the code prior to these changes, but I can't really see what it's doing there either. Any ideas?

Reply:
The `clean_paths` method above turns it into an array. For example, if the value is `path1,path2` it'll get parsed into `%w(path1 path2)`.

This allows the controller to support the old style URLs (with the paths given directly in the URL parameters) and the new stlye URLs (where the paths is an ID of the saved paths).

Reply:
Hmm, I don't think I understand the old and new style URLs. So to clarify, the `paths` query parameter is either used for a list of paths as you describe, or as a single 10 character identifier, and the code here is assuming that it's the latter?

Reply:
Yep, that's correct.